scripts/background.js

```diff
@@ -0,0 +1,13 @@
+chrome.tabs.onUpdated.addListener((tabId, changeInfo, tab) => {
+  const tabUrl = tab.url ?? tab.pendingUrl;
+  if (
+    changeInfo.status === 'complete' &&
+    tabUrl &&
+    tabUrl.includes('amazon.com')
+  ) {
+    chrome.scripting.insertCSS({
+      target: { tabId: tabId },
```

Comment:
Grabs the tab id and injects the global css file. Not sure if this code is actually necessary though since we're loading it with the browser extension.